### DISCUSSION TITLE
Error: Could not add the blog

### DISCUSSION THREAD

**Discussion by ansgar68** [TumblThreeApp/TumblThree#310] [Q&A (Question & Answer)]
I cannot add any blogs, I keep getting the error message:
ERROR: Could not add the blog. The url https://www.tumblr.com/blog/view/planesawesome cannot be recognised as valid blog!

I am running Tumblthree 2.5.5 X64 on Windows10 64bit
It's on my Data drive with no write restrictions.
Tried running as admin which didn't do anything.

Any idea what could help?

**Comment by hrxn** [comment]
Tried adding `https://planesawesome.tumblr.com`?

**Comment by thomas694** [comment]
seems to be solved

**Comment by thomas694** [comment]
The issue has been closed. You can still comment. Feel free to ask for reopening the issue if needed.
